statsmodels/stats/tests/test_power.py

```diff
@@ -727,28 +727,6 @@ def test_power_solver_warn():
     nip.start_bqexp['nobs1'] = {'upp': 50, 'low': -20}
     val = nip.solve_power(0.1, nobs1=None, alpha=0.01, power=pow_, ratio=1,
                           alternative='larger')
-    import scipy
-    if scipy.__version__ < '0.10':
```

Comment:
I don't know if I make myself clear:
Tests currently pass because those tests are not enabled.
We need to remove the `if scipy.__version__ ...` condition, unindent the following code an check whether those tests no pass on recent scipy.
https://github.com/bashtage/statsmodels/commit/f33ebfb9f599a53bce507847e9a2a3712dab80cd

There are several backup approaches to solve for the root, and different scipy versions succeeded or failed in different cases with different optimize solvers/root finders. 
But this code check has never been updated since I wrote it for scipy 0.9